tests/e2e/test_rosacli_upgrade.go

```diff
@@ -392,8 +392,6 @@ var _ = Describe("Cluster Upgrade testing",
 			By("Check upgrade state")
 			err = WaitForUpgradeToState(upgradeService, clusterID, con.Scheduled, 4)
 			Expect(err).To(BeNil())
-			err = WaitForUpgradeToState(upgradeService, clusterID, con.Started, 70)
```

Comment:
We don't wait for upgrade started anymore ?

Reply:
If we will wait, the upgrade started will never be cancelled/deleted for next test cases.

Reply:
In the test case I see `Check the upgrade is finished successfully`
So we should add a step to wait for the upgrade

Reply:
Okay will try to mould the test case same